pkg/test/echo/cmd/client/main.go

```diff
@@ -142,6 +140,12 @@ func init() {
 		"Treat as a server first protocol; do not send request until magic string is received")
 	rootCmd.PersistentFlags().BoolVarP(&followRedirects, "follow-redirects", "L", false,
 		"If enabled, will follow 3xx redirects with the Location header")
+	rootCmd.PersistentFlags().BoolVar(&newConnectionPerRequest, "new-connection-per-request", false,
```

Comment:
If you added this to the proto, why does it need a flag ? It's a bit confusing - there is a default value in proto, but also one in flag - I doubt anyone will understand how they interact. 

What is the current behavior ? For gRPC it seems a new connection is used by default, so it would change the behavior.

Reply:
This client app is just a main wrapper around the forwarder. This used to be the way that echo worked ... we would exec onto the pod and then run this command. TBH I'm not really sure if it's used anymore. This is just updating the app to allow configuring the new parameter. This is consistent with how all of the parameters are set by this app.